servers/quarkus-server/build.gradle.kts

```diff
@@ -157,6 +157,10 @@ tasks.withType<Test>().configureEach {
   systemProperty("quarkus.smallrye.jwt.enabled", "true")
   // TODO requires adjusting the tests - systemProperty("quarkus.http.test-port", "0") -  set this
   //  property in application.properties
+  systemProperty(
```

Comment:
Does this really allow deverlopers to override the version??

Reply:
yes, it doesn't allow overriding. But I don't see any other way of doing it. Any suggestions?